nanopb.podspec

```diff
@@ -1,6 +1,7 @@
 Pod::Spec.new do |s|
   s.name         = "nanopb"
-  s.version      = "0.3.9011"
+  # CocoaPods minor version is minor * 100,000 + patch * 100 + fourth
+  s.version      = "1.300905.0"
```

Comment:
Leaving aside the math for the moment, this looks weird: it allows for ~unlimited minor versions, 1000 patch versions, and 100 fourth versions. This would look more regular if we allowed for the same number of patch and fourth versions (either 100 or 1000).

Note that if we start with 100, we can always expand this later. (Pushing out the number of digits will make the minor version much larger than the previous one.